aws/ci_build.rake

```diff
@@ -223,8 +219,8 @@ def upgrade_frontend(name, host)
     HipChat.log "<b>#{name}</b> (#{host}) failed to upgrade, removing from rotation.", color: 'red'
```

Comment:
Don't we need to re-register the frontend with the ELB after it is successfully started?